src/lib/ngsi/Metadata.cpp

```diff
@@ -174,3 +174,46 @@ void Metadata::fill(const struct Metadata& md)
   value       = md.value;
   association = md.association;
 }
+
+
+
+/* ****************************************************************************
+*
+* renderV2 - 
+*/
+std::string Metadata::renderV2(ConnectionInfo* ciP, RequestType request, bool comma)
```

Comment:
My opinion is based on the understanding that ciP and RequestType is needed when you need to know tag names and this kind of stuff in the case of XML. But, in the case of a pure JSON render, are they actually needed? If they are needed, they should be keep; if they are not needed, they should be removed.